modules/gob-web/modules/student/student-summary.js

```diff
@@ -44,6 +45,15 @@ const welcomeMessages = [
 	'halo, ', // indonesian
 	'Salve, ', // latin
 	'Χαῖρε! ', // ancient greek
+	'Zdravo, ', // bosnian
+	'Bok, ', // croatian
+	'ahoj, ', // czech, slovak
+	'Tere, ', // estonian
+	'Bula, ', // fijian
+	'Zdravo, ', // serbian
+	'Hujambo, ', // swahili
+	'Xin chào, ', // vietnamese
```

Comment:
Is this the right comma for Vietnamese? I know Korean/Japanese/Chinese all use non-english commas, if they use commas at all.

Reply:
Everywhere I look points to this comma.